paymentsheet/build.gradle

```diff
@@ -92,4 +100,23 @@ dependencies {
     androidTestImplementation "androidx.test:runner:$androidTestVersion"
     androidTestUtil "androidx.test:orchestrator:$androidTestVersion"
 
+    ktlint "com.pinterest:ktlint:$ktlintVersion"
+}
+
+task ktlint(type: JavaExec, group: "verification") {
+    description = "Check Kotlin code style."
+    main = "com.pinterest.ktlint.Main"
+    classpath = configurations.ktlint
+    args "src/**/*.kt"
+    // to generate report in checkstyle format prepend following args:
+    // "--reporter=plain", "--reporter=checkstyle,output=${buildDir}/ktlint.xml"
+    // see https://github.com/pinterest/ktlint#usage for more
+}
+check.dependsOn ktlint
+
+task ktlintFormat(type: JavaExec, group: "formatting") {
```

Comment:
sorry missed this one, probably add this in in [pre-push](https://github.com/stripe/stripe-android/blob/master/.githooks/pre-push) as well?

Reply:
I think the ktlint in that pre-push will cover it.

Reply:
Right now it runs only on `payments-core`.

https://github.com/stripe/stripe-android/blob/ad426ca33105e7bc5f17c027f8146354d13f75ae/.githooks/pre-push#L14